pkgs/top-level/all-packages.nix

```diff
@@ -23267,6 +23267,13 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
   };
 
+  mongodb-6_0 = callPackage ../servers/nosql/mongodb/6.0.nix {
+    sasl = cyrus_sasl;
```

Comment:
```suggestion
```
We might as well remove this. Also we should drop some older mongodb versions.

Reply:
The `sasl` input still seems to be needed (forwaded from `6.0.nix` to
`mongodb.nix`).

For the older MongoDB versions, many seem to be currently broken on master.
They are being fixed in #172009.

I'm not sure of which versions should be dropped, but I think this should be
done in a subsequent pull request.